CMakeLists.txt

```diff
@@ -501,6 +505,11 @@ while(library_variants_copy)
 
     set(runtimes_flags "${common_flags} -ffunction-sections -fdata-sections -fno-ident --sysroot ${install_directory}")
 
+    set(compiler_rt_extra_flags)
+    if(flags MATCHES "-mfloat-abi=hard")
+        set(compiler_rt_extra_flags "${compiler_rt_extra_flags} -DCOMPILER_RT_ARMHF_TARGET")
```

Comment:
A brief comment please explaining what this does, ideally with a link to docs for COMPILER_RT_ARMHF_TARGET

Reply:
Thanks. In the course of writing the comment explaining why I couldn't do this a nicer way, I found a nicer way :-)